js_modules/dagit/packages/core/src/app/LayoutProvider.tsx

```diff
@@ -49,12 +31,12 @@ export const LayoutProvider: React.FC = (props) => {
     () => ({
       nav: {
         isOpen: navOpen,
-        isCollapsible: isInstancePage || isSmallScreen,
+        isCollapsible: true,
```

Comment:
I think if this is always true, there is more code that can be cleaned up in `App` and `AppTopNav`. E.g. the hamburger icon should always be visible, the CSS can be simplified.